src/.htaccess

```diff
@@ -84,7 +84,6 @@ redirect permanent /x https://xhr.spec.whatwg.org/
 redirect permanent /xhr https://xhr.spec.whatwg.org/
 
 redirect permanent /newbug https://github.com/whatwg/html/issues/new
-redirect permanent /faq https://wiki.whatwg.org/wiki/FAQ
```

Comment:
User agents tend not to cache them long so I think we should be fine. We could go with frequently-asked-questions though if we wanted to. Would be consistent with code-of-conduct and working-mode.